src/main/java/uk/gov/companieshouse/siccode/api/search/SicCodeController.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.companieshouse.siccode.api.search;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+import uk.gov.companieshouse.siccode.api.SicCodeApiApplication;
+
+/**
+ * SicCodeController
+ */
+
```

Comment:
Remove blank line between Javadoc & class.
Actually, is there any point in the Javadoc?

Reply:
See also SicCodeService